examples/hello-world/package.json

```diff
@@ -10,6 +10,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
```

Comment:
Should we be adding these as dependencies?

I followed Nextjs on this which makes the user instal it themselves

Reply:
Hmm I wonder if we should just install `react react-dom react-router` for users or if it's better to keep them as peer dependencies

nextjs has react as a peer dependency, whereas nuxtjs has vue as a dependency


either way this is what nextjs uses:

```
	  "standard": {
    "parser": "babel-eslint",
    "ignore": [
      "**/node_modules/**",
      "**/examples/**/lib/**"
    ]
  }
```

and btw they use single quotes and no semi colons so lets us whatever they're using!